tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ find_package(GTest ${NOVELRT_GTEST_VERSION} REQUIRED
   COMPONENTS gtest_main
 )
 
+set(INTEROPBUILD 0)
```

Comment:
Wat?

Reply:
Just use NOVELRT_BUILD_INTEROP? `$<$<BOOL:${NOVELRT_BUILD_INTEROP}>:Interop>`

Reply:
Yup - switched this since I didn't know about the BOOL treatment here